### ISSUE TITLE
Terminal overflows screen on 1.1.0

### ISSUE THREAD

**Issue by 0atman** [GitSquared/edex-ui#291]
New bug on v1.1.0, happened as soon as I upgraded, just now.

System: Linux (Ubuntu 18.04)
Shell: zsh

![screenshot from 2018-11-25 20-18-11](https://user-images.githubusercontent.com/114097/48984117-9441f280-f0ef-11e8-95b0-4afc2fdb9efb.png)

**Comment by 0atman** [comment]
Every time!

**Comment by kevinwainczak** [comment]
It overflows horizontally as well. 
<img width="1096" alt="screen shot 2018-11-25 at 4 17 31 pm" src="https://user-images.githubusercontent.com/14118162/48984736-a6129e00-f0cd-11e8-9263-388449b92656.png">

**Comment by kevinwainczak** [comment]
@GitSquared I'm getting a result saying that `window.term.fit()` is not a function. `window.term` is successfully recognizing the terminal, though.

**Comment by GitSquared** [comment]
@kevinwainczak Crap, of course. How about `window.term[window.currentTerm].fit()` ?

**Comment by eadwu** [comment]
Can confirm on NixOS on master and 1.1.0. Other things I notice are that clicking on a tab other than the main shell more than once (after spawning the shell) causes malformation of the entire terminal window. Also trying to spawn a tab during the loading process makes it eventually crash with `ioctl` error (this might be a problem with my build process).

```
✖  fatal     Error: ioctl(2) failed.
    at UnixTerminal.resize (/nix/store/0d8n97ivx5b1w9a165gws895jvy6c7am-eDEX-UI-bare-1.1.0/src/node_modules/node-pty/lib/unixTerminal.js:213:13)
    at EventEmitter.Terminal.Ipc.on.args (/nix/store/0d8n97ivx5b1w9a165gws895jvy6c7am-eDEX-UI-bare-1.1.0/src/classes/terminal.class.js:312:34)
    at EventEmitter.emit (events.js:187:15)
    at WebContents.<anonymous> (/nix/store/dn245bsxp2hdyz6njwcr4dq7gl3mxvlj-electron-3.0.10/lib/electron/resources/electron.asar/browser/api/web-contents.js:284:13)
    at WebContents.emit (events.js:182:13)
```

**Comment by GitSquared** [comment]
@eadwu Yeah, i got that one under control. Fix on the way.

**Comment by kevinwainczak** [comment]
@GitSquared Calling the function doesn't fix it, but the function is getting called.

**Comment by GitSquared** [comment]
@kevinwainczak Damned. This is straight from xterm.js, i don't see why it stopped working.

Thanks for your help.

**Comment by 0atman** [comment]
amazing hustle guys, thank you!

**Comment by Cry0lab** [comment]
This is happening for me as well, Fedora 28, if that matters.

**Comment by GitSquared** [comment]
Fix coming tomorrow i'm afraid.

**Comment by kingfengji** [comment]
same here, windows 10. 
But still awesome and badass upgrade, nice to have multiple tabs yo.
![bug](https://user-images.githubusercontent.com/3418996/48990164-5503b800-f168-11e8-9f72-dfb00beffcf7.PNG)

**Comment by martin-flaregames** [comment]
Running into the same issue, using Manjaro / XFCE / bash. Installed from AUR `edex-ui` (1.1.0-1)

**Comment by warsix4** [comment]
Windows 10 v1809 same issue. Installed from installer exe.

**Comment by GitSquared** [comment]
![screenshot from 2018-11-26 18-42-17](https://user-images.githubusercontent.com/24496417/49031890-7f2a9980-f1ab-11e8-81eb-93b05b0ae473.png)

Now i see why i didn't get this issue sooner. For some weird web-engine related reason this bug occurs only on certain themes. Go figure. I'll see if i can figure a hack that'll make this work again.

**Comment by GitSquared** [comment]
@0atman @kevinwainczak @eadwu @Cry0lab @kingfengji @martin-flaregames @warsix4 

Thank you all for the reports. Issue has been fixed & i'm releasing v1.1.1 to include this along with the fix for #292 and one for a theme-changer related issue. Original v1.1.0 assets will be overwritten so the broken release can't be downloaded again.

Sorry for the delay!

**Comment by kingfengji** [comment]
Works for windows here, dude.
Plus: the chalkboard theme supports keywords highlight with different color! 
I'm on fire!
